starport_template/lib/entities/transaction.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:starport_template/entities/amount.dart';
+import 'package:starport_template/entities/denom.dart';
+
+enum TransactionType {
+  Receive,
+  Send,
+}
+
+class Transaction {
```

Comment:
how much different is it from the `MsgSendTransaction`? could we somehow reuse it here? or choose a different naming that explains the difference between `Transaction` and `MsgSendTransaction`

Reply:
The only common thing between the two is the `amount` parameter. The `Transaction` is something which is populated for transactions history so it's quite different from `MsgSendTransaction`

Reply:
We can name it `TransactionHistory` maybe

Reply:
`TransactionHistoryItem`?